<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>managerpage</class>
 <widget class="QMainWindow" name="managerpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#managerpage{
border-image: url(:/picture2.jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="cardTable">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>621</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 244, 244);</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="controlPage">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>130</y>
      <width>321</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(244, 244, 244);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="lineWidth">
       <number>4</number>
      </property>
      <property name="text">
       <string>  Add new project</string>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_addcreate">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>330</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Create!</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>291</width>
        <height>205</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>MV Boli</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>projectid</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="setid">
           <property name="placeholderText">
            <string>add id</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>MV Boli</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>projectname</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="setname">
           <property name="placeholderText">
            <string>add name</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>MV Boli</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>startdate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="setstart">
           <property name="placeholderText">
            <string>add startdate</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>MV Boli</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>enddate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="setend">
           <property name="placeholderText">
            <string>add enddate</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>MV Boli</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>status</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="setstat">
           <property name="placeholderText">
            <string>add status</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Alter project&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_altercreate">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>330</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Alter!</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>291</width>
        <height>205</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <family>MV Boli</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>projectid</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="chooseid">
           <property name="placeholderText">
            <string>alter id</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <family>MV Boli</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>projectname</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="newname">
           <property name="placeholderText">
            <string>alter name</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <family>MV Boli</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>startdate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="newstart">
           <property name="placeholderText">
            <string>alter startdate</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="font">
            <font>
             <family>MV Boli</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>enddate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="newend">
           <property name="placeholderText">
            <string>alter enddate</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="font">
            <font>
             <family>MV Boli</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>status</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="newstat">
           <property name="placeholderText">
            <string>alter status</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Delete project</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_deletecreate">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>320</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Delete!</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>120</y>
        <width>291</width>
        <height>166</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="font">
            <font>
             <family>MV Boli</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>projectid</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="deleteid">
           <property name="placeholderText">
            <string>delete id</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="font">
            <font>
             <family>MV Boli</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>projectname</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="deletename">
           <property name="placeholderText">
            <string>delete name</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="font">
            <font>
             <family>MV Boli</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>status</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="deletestatus">
           <property name="placeholderText">
            <string>delete status</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>50</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_alter">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>50</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Alter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>50</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_search">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchid">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>300</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>31</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 244, 244);</string>
    </property>
    <property name="placeholderText">
     <string>  Search some ids...</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::CursorMoveStyle::VisualMoveStyle</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_logout">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>530</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Log out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_count">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Count</string>
    </property>
   </widget>
   <widget class="QLabel" name="rowcount_project">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>67</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>969</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
